nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/test/resources/test-invite-contrib.xml

```diff
@@ -12,6 +12,9 @@
   <extension target="org.nuxeo.ecm.user.invite.UserInvitationService"
              point="configuration">
 
+    <configuration remove="true">
+    </configuration>
+
```

Comment:
i don't get why this is needed: is it because the component does not handle override?